src/pages/AboutPage.js

```diff
@@ -117,6 +118,37 @@ const AboutPage = () => (
     <hr />
 
     <Row className="about-align-center">
+      <Col className="about-order1">
+        <h2>Build for Black Lives in the News</h2>
+        <p> Check out these latest news articles featuring Build for Black Lives to learn
+            more about the motivation and vision behind this platform.
+        </p>
+        <p className="about-list">
+          <p>
+
+            <a href="https://news.northeastern.edu/2020/08/10/can-technology-reduce-racial-inequity-build-for-black-lives-a-student-built-platform-could-help/">
```

Comment:
add a target __blank

Reply:
for the other links on this page too

Reply:
While you're at it: add `rel="noopener noreferrer"`

https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/